docs-client/webpack.config.ts

```diff
@@ -30,6 +30,7 @@ const config: Configuration = {
     path: path.resolve(process.cwd(), './build/web'),
     // We don't mount to '/' for production build since we want the code to be relocatable.
     publicPath: isDev ? '/' : '',
+    filename: isDev ? '[name].js' : '[name].[contenthash].js'
```

Comment:
Question: Can't this be just `'[name].[contenthash].js'`?

Reply:
When hot loading is done, the file contents and hence the hash would change.
I thought this would be inconvenient when debugging, but do you prefer that we just use `contenthash`?

Reply:
No, I just wanted to know if we could just use the file name with the hash.
Thanks!